resources/assets/js/videos/components/videoScreen/drawInteractions.vue

```diff
@@ -163,6 +163,19 @@ export default {
             this.$emit('pending-annotation', null);
         },
         extendPendingAnnotation(e) {
+            // Check polygons
+            if (e.feature.getGeometry().getType() === 'Polygon') {
+                if (PolygonValidator.isInvalidPolygon(e.feature)) {
+                    // Disallow polygons with less than three non-overlapping points
+                    this.$emit('is-invalid-polygon')
+                    this.removeFeature(e.feature)
```

Comment:
This doesn't work. The `this.removeFeature` below was executed inside an event handler of `pendingAnnotationSource`, so the `this` referred to the source there.

I'll fix this myself while I'm doing my final review.